website/scripts/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+MMVERSION=${1-0.3.35}
+
+echo Running website in container...
+WEBSITE_CID=$(docker run \
+    --detach \
+    --rm \
+    --publish "4567:4567" \
+    --publish "35729:35729" \
+    --volume "$PWD:/website" \
+    hashicorp/middleman-hashicorp:${MMVERSION})
+echo Website running in container ${WEBSITE_CID}.
+
+trap 'echo Killing docker website container... && docker kill ${WEBSITE_CID}' EXIT HUP INT QUIT TERM
```

Comment:
curious on the use of trap here instead of capturing the return from wget and then stopping the container at the end of this script.

Reply:
probably overkill for a single test step (wget)... i'll change it for simplicity